tests/cpp/test_resize.cpp

```diff
@@ -2000,7 +2001,8 @@ TEST_F(ResizeTest, ResizeReshapeAndSlice) {
       tv1,
       {{IrBuilder::create<Val>(0L), IrBuilder::create<Val>(2L)},
        {IrBuilder::create<Val>(0L), IrBuilder::create<Val>(2L)}});
-  fusion->addOutput(tv2);
+  auto tv3 = add(tv2, tv2);
```

Comment:
So MarkAliasesPreparePass doesn't kick in. Other changes in the file are minor cleanups.

Reply:
can we add this comment to the code?